maketarget/loadprimarytarget.py

```diff
@@ -28,14 +28,15 @@ def execute(self, context):
         sks = obj.data.shape_keys
         pt = sks.key_blocks["PrimaryTarget"]
```

Comment:
(Comment written just to demonstrate what happens)
Maybe we should use better variable names.